### ISSUE TITLE
[addtool] pwnat

### ISSUE THREAD

**Issue by gwen001** [gwen001/offsectools_www#1332]
[tags]firewall,nat,pivot[/tags]
[short_descr]Punch holes through firewalls/NATs where both clients and servers can be behind separate NATs.[/short_descr]
[link] https://samy.pl/pwnat/ [/link]
[link] https://github.com/samyk/pwnat [/link]
[long_descr]
pwnat, by Samy Kamkar, is a tool that allows any client behind a NAT to communicate with a server behind a separate NAT with no port forwarding and no DMZ setup on any routers in order to directly communicate with each other.

There is no middle man, no proxy, no third party, no UPnP required, no spoofing, no DNS tricks. The server does not need to know the client's IP address before connecting.

More importantly, the client can then connect to any host or port on any remote host or to a fixed host and port decided by the server.

Simply put, this is a proxy server that works behind a NAT, even when the client is also behind a NAT.
[/long_descr]
[image] https://raw.githubusercontent.com/gwen001/offsectools_www/main/tmp/d7eda8b2d53a1615101f1569812e1eb0.png [/image]

**Comment by gwen001** [comment]
Issue correctly handled, tool is waiting for human validation.

**Comment by gwen001** [comment]
Tool has been accepted by the team: https://offsec.tools/tool/pwnat

Thank you for your contribution!
